src/Header/RequestHandler.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_REQUESTHANDLER_H_
+#define GROUP4_HEADER_REQUESTHANDLER_H_
+
+#include "OperateData.h"
+#include "Authenticator.h"
+#include "APIHandler.h"
+#include <string>
+#include <unordered_map>
+#include <memory>
+
+//Will decode the request string and check if it's a valid request.
+//calling RequestParser to get OperateData then call correspond api function
+class RequestHandler
+{
+public:
+    //default constructor will never be used
+    RequestHandler();
```

Comment:
If it's not used, then please delete it.

Reply:
Done